internal/audit/local/badgerdb.go

```diff
@@ -76,52 +81,66 @@ func NewLog(conf *Conf, decisionFilter audit.DecisionLogEntryFilter) (*Log, erro
 	maxBatchSize := int(conf.Advanced.MaxBatchSize)
 	ttl := conf.RetentionPeriod
 
+	ticker := time.NewTicker(flushInterval)
+	defer ticker.Stop()
```

Comment:
🤦‍♂️

Reply:
Annoyingly, I need access to it in the `cerboshub` case as I reset it's interval on each sync in `batchSyncer`. So I'd need to initialize outside, and `Close()` inside 😕. The whole thing was only introduced to allow synchronous triggers to prevent the need to `Sleep` in tests. Wonder if it's worth the hassle.